From 5bdad46ac163562d141537b94f2eb450ea937c2c Mon Sep 17 00:00:00 2001
From: James Page <james.page@chainguard.dev>
Date: Thu, 14 Aug 2025 13:39:47 +0100
Subject: [PATCH] airflow-provider-slack: avoid secret false positive

trivy will detect the Slack access token as a potential secret,
generating a HIGH vulnerabiluty when present in an OCI image.

Replace with a reminder instead.
---
 providers/slack/src/airflow/providers/slack/hooks/slack.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/providers/slack/src/airflow/providers/slack/hooks/slack.py b/providers/slack/src/airflow/providers/slack/hooks/slack.py
index 7ea38a0310..e09ff44c14 100644
--- a/providers/slack/src/airflow/providers/slack/hooks/slack.py
+++ b/providers/slack/src/airflow/providers/slack/hooks/slack.py
@@ -400,7 +400,7 @@ class SlackHook(BaseHook):
                 "password": "Slack API Token",
             },
             "placeholders": {
-                "password": "xoxb-1234567890123-09876543210987-AbCdEfGhIjKlMnOpQrStUvWx",
+                "password": "REPLACE ME WITH A SLACK ACCESS TOKEN",
                 "timeout": "30",
                 "base_url": "https://www.slack.com/api/",
                 "proxy": "http://localhost:9000",
-- 
2.43.0

